nwg_panel/modules/sway_workspaces.py

```diff
@@ -114,9 +106,47 @@ def build_number(self, num, label):
 
         return eb, lbl
 
+    def get_ws_label(self, num, idx = None, ws_defs = None, focused = False):
+        """
+        Get the text to display for a workspace:
+        - num: the number of the workspace (as integer)
+        - idx: the index of the workspace in ws_defs (if defined)
+        - focused: if the workspace is currently focused
+        - ws_defs: as returned by find_details()
+        """
+
+        # config_idx is the index of the current workspace in the
+        # configuration. It is the index of the workspace in the
+        # numbers config field, or the an index based on the
+        # workspace number, first workspace being number 1
+        config_idx = None
```

Comment:
`settings["numbers"]` is stored as a list of *str*, converting to *int* will fail the lookup here